doc/source/ray-overview/pip_freeze_ray-ml-py39-cpu.txt

```diff
@@ -352,7 +352,7 @@ PyWavelets==1.4.1
 PyYAML==6.0.1
 pyzmq==24.0.1
 querystring-parser==1.2.4
-ray @ file:///home/ray/ray-3.0.0.dev0-cp39-cp39-manylinux2014_x86_64.whl#sha256=817c9218d57cb6101cdcbcd209aa7b17960a55626fb48763c7077ed36508d3eb
+ray @ file:///home/ray/ray-2.8.0-cp39-cp39-manylinux2014_x86_64.whl#sha256=817c9218d57cb6101cdcbcd209aa7b17960a55626fb48763c7077ed36508d3eb
```

Comment:
this won't work, it is a freeze file with a sha hash (I do not know how this is generated actually. please revert this, and the other doc/freeze txt one.

you probably needs to consult the folks who generated this on how to regenerate this.

the `file://` does not really make sense to me.

cc @can-anyscale

Reply:
reverted for now (same with below)